test/webapi/ows/stac/demo-collection.json

```
@@ -7,6 +7,19 @@
       ],
       "type": "application/zarr",
       "href": "http://localhost:8080/s3/datasets/demo.zarr",
+      "xcube:store_kwargs": {
```

```suggestion
      "xcube:data_store_params": {
```

to align it with keyword argument `data_store_params` of the data store API function `new_data_store()`.